### PR TITLE
LPD-27271 Add a DM Folder in French language

### PR DESCRIPTION
**Pull request by ambrinchaudhary**
# What is this trying to solve?

[Link to ticket](https://issues.liferay.com/browse/LPD-27271)

DM Folders cannot be created in french language. Console errors appear. 

# How am I fixing it?

There was a conflict in single quotes with a key message that also contained a single quote. 
I am escaping the message and using doble quotes to avoid this. 

For the test, I finally decided to not use current Pages for Documents and Media, as all the locators change from English to French, and the code was going to be less readable. So I just did everything in the test itself. 
Screenshot 2024-06-03 at 15 14 15

# How can you verify that it works?

Follow the steps, or run the provided unit test: 

1. Change the URL from http://localhost:8080/home to http://localhost:8080/fr/home
2. Open Product Menu → Content → Documents & Media → New → Folder

### CODE REVIEW

**Comment by ambrinchaudhary** [comment]
cc/ @liferay-frontend

**Comment by ambrinchaudhary** [comment]
ci:test:relevant

**Comment by ambrinchaudhary** [comment]
ci:test:sf

File: portal-web/docroot/html/taglib/ui/input_permissions/page.jsp [hunk 1]
```diff
@@ -172,7 +172,7 @@ String modelName = (String)request.getAttribute("liferay-ui:input-permissions:mo
 				var inputPermissionsOptionsButton = document.getElementById('<%= uniqueNamespace %>inputPermissionsOptionsButton');
 
 				if (inputPermissionsOptionsButton) {
-					inputPermissionsOptionsButton.innerText = force ? '<%= LanguageUtil.get(request, "hide-options") %>' :'<%= LanguageUtil.get(request, "more-options") %>' ;
+					inputPermissionsOptionsButton.innerText = force ? '<%= StringUtil.replace(LanguageUtil.get(request, "hide-options"), "\'", "\\'") %>' : '<%= StringUtil.replace(LanguageUtil.get(request, "more-options"), "\'", "\\'") %>' ;
```
Comment (adolfopa) [inline portal-web/docroot/html/taglib/ui/input_permissions/page.jsp:175]: Shouldn't `HtmlUtil.escapeJs` be used here?

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: portal-web/docroot/html/taglib/ui/input_permissions/page.jsp (hunk 1 above)
Comment (ambrinchaudhary) [reply portal-web/docroot/html/taglib/ui/input_permissions/page.jsp]: You are right, that works as well. I've updated the pull. Thanks!

**Review by DiegoHu97 — APPROVED** [review]
Works fine!

**Comment by ambrinchaudhary** [comment]
ci:forward

**Comment by ambrinchaudhary** [comment]
ci:test:report:11414776

**Comment by ambrinchaudhary** [comment]
ci:report:11414776

**Comment by ambrinchaudhary** [comment]
No unique failures in this pull. I will force forward

**Comment by ambrinchaudhary** [comment]
ci:force:forward

**Comment by ambrinchaudhary** [comment]
ci:forward:force

**Pull request closed without merging**